### ISSUE TITLE
Compartilhar Projeto no GitHub

### ISSUE THREAD

**Issue by nellefc** [cami-la/sacola-api_IFOOD_DEV_WEEK#37]
Oi Cami, tudo bem?

Gostaria de aprender a compartilhar o projeto no GitHub. Este do iFood Dev Week, poderia me ajudar?

Obrigada!

**Comment by cami-la** [comment]
Claro, @nellefc. Vou Fazer uma mentoria daqui a 15 minutos sobre POO e vou compartilhar o projeto no GitHub.
Vamos lá que vou fazer ao vivo: https://web.dio.me/lives/dominando-programacao-orientada-a-objetos-com-java-ifood

**Comment by nellefc** [comment]
Maravilha!!! Já estou assistindo!!! 😊

**Comment by cami-la** [comment]
Oie, boa tarde. (:
E aí, conseguiu subir o projeto no GitHub?

**Comment by nellefc** [comment]
Oie!

Se quiser ver se está ok, o link é

<https://github.com/nellefc/Sacola_API_iFood_Dev_Week>
[Sacola_API_iFood_Dev_Week.png]
nellefc/Sacola_API_iFood_Dev_Week: Código-Fonte do Projeto Sacola API do iFood Dev Week<https://github.com/nellefc/Sacola_API_iFood_Dev_Week>
github.com<https://github.com/nellefc/Sacola_API_iFood_Dev_Week>

Beijos 😘

Em 13 de out. de 2022, à(s) 14:53, Camila Cavalcante ***@***.***> escreveu:

﻿

Oie, boa tarde. (:
E aí, conseguiu subir o projeto no GitHub?

**Comment by cami-la** [comment]
Ficou muito bom, @nellefc.
Parabéns! 👏👏🥲

**Comment by nellefc** [comment]
Graças a minha professora top!!! 😊👏🏼

Enviado do meu iPhone

Em 14 de out. de 2022, à(s) 12:30, Camila Cavalcante ***@***.***> escreveu:

﻿

Ficou muito bom, @nellefc<https://github.com/nellefc>.
Parabéns! 👏👏🥲
